app/views/users/_sidebar.html.erb

```diff
@@ -8,7 +8,7 @@
       <% pseud_link_text = current_page?(@user) ? ts("Pseuds") : (@author ? @author.name : @user.login) %>
       <a href="#" title="<%= ts("Pseud Switcher") %>"><%= pseud_link_text %></a>
       <ul class="expandable secondary">
-        <%= print_pseud_selector(@user.pseuds) %>
+        <%= print_pseud_selector(@user.pseuds.for_profile) %>
```

Comment:
The pseud selector is set up as a sequence of `span_if_current` links. This means that if the user is viewing a particular pseud's dashboard and they view the selector in the sidebar, the current pseud is highlighted in pale grey. But always displaying the first 20 pseuds means that when the user is viewing a particular pseud, they're not guaranteed to see it in the sidebar. Is that okay?

(Should the current pseud be added to the list at the top? Should the list of pseuds be paginated and show the page that the pseud would be on?)

Reply:
Putting the currently-selected pseud at the top sounds good.